docs/scripts/formattedTSDemos.js

```diff
@@ -55,7 +55,8 @@ async function getFiles(root) {
           files.push(...(await getFiles(filePath)));
         } else if (
           stat.isFile() &&
-          /\.tsx?$/.test(filePath) &&
+          (/\.tsx?$/.test(filePath) || /\.ts?$/.test(filePath)) &&
+          !filePath.endsWith('types.ts') &&
```

Comment:
We should never convert to JS type files, not sure what would be the best way to detect this, for now I just test that the file doesn't end with `types.ts`.

Reply:
The issue was that the file was not empty but looks like this:

```
export default {}

```

I didn't want to check for this as a string 😅 . Yeah, let's keep the current one, we can update it in the future if needed.